### DISCUSSION TITLE
How can I look up the domain that Jupyterlite is running in?

### DISCUSSION THREAD

**Discussion by epugh** [jupyterlite/jupyterlite#777] [Q&A] [answered]
I have Jupyterlite deployed as part of my webapp, Quepid.   Quepid has api's that Jupyterlite can access:

```
# Retrieve from Quepid API all the snapshots
snapshots = []
for snapshot_id in SNAPSHOT_IDS:
    res = await fetch(f'http://localhost:3000/api/cases/{CASE_ID}/snapshots/{snapshot_id}.json')
    snapshots.append(await res.json())
```

How can I figure out what domain I am in, and or/ do a relative URL in `fetch()`?

**Comment by epugh** [comment, marked as answer]
Shockingly, the relative URL did work!!!!   

```
res = await fetch(f'/api/cases/{CASE_ID}/snapshots/{snapshot_id}.json')
```

I am pumped about how useful Jupyterlite is going to be for folks doing relevancy tuning with Quepid!    (Learn more at http://www.quepid.com)

**Reply by jtpio** [reply]
Nice, thanks for sharing the solution!
